sdk/resourcemanager/Azure.ResourceManager/tests/Azure.ResourceManager.Tests.csproj

```diff
@@ -8,6 +8,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Identity" />
     <PackageReference Include="Microsoft.Extensions.Azure" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" />
```

Comment:
@JonathanCrd :  Something is still attempting to pull in a down-level version of this package, causing the abstractions conflict.  

We should look at pulling up the rest of the extensions-specific package versions and see if we can isolate what was causing that issue.

Reply:
Is this related to the NuGet cache issue you mentioned?

Reply:
No, that was why my local build was cranky.   Something in the tree is pulling in this package at a lower version, causing a mismatch.   I added this here to force the version upgrade until we can figure out what.